src/common/ui-components/Menu/Menu.module.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../styles/icons';
```

Comment:
hi! currently the only comment I have give to you is that it is great, but I am not happy about the magic numbers all over the CSS. Please, get a habit of extracting the numerical values into variables. Thanks!